### ISSUE TITLE
Problem Forking

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/gogs#75]
After following these instructions:
- Go to https://git.door43.org/Door43/SourceTextRequestForm and click the “Fork” button in the upper right.
- Once that is completed, go to https://git.door43.org/chrisjarka/SourceTextRequestForm/src/master/.github/ISSUE_TEMPLATE.md and edit the file (paste in your wanted text).
- Once that is done and saved, go to https://git.door43.org/Door43/SourceTextRequestForm/pulls and click “New Pull Request"

Christine got the following error:
![image](https://cloud.githubusercontent.com/assets/107731/18291692/fbfcdbc4-7446-11e6-8c27-b1166416453b.png)

**Comment by ethantkoenig** [comment]
I can't reproduce, maybe it's been fixed?
